scripts/new_project.kts

```diff
@@ -4,7 +4,16 @@ object NewProject {
 
     private const val ARGUMENT_DELIMITER = "="
     private const val DOT_SEPARATOR = "."
+    private const val HELP_MESSAGE = """
+Run kscript new_project.kts to create a new project with the following arguments:
+    package-name=   New package name (i.e. com.example.package)
+    app-name=       New app name (i.e. MyApp, "My App")
+    --help          Display useage message and exit
```

Comment:
I believe this should be "Display **usage** message and exit"

Reply:
@chornerman it is the message for `--help` option when we use it like `kscript new_project.kts --help` to print out how we can use `new_project.kts` to make a sample project.

Reply:
@kaungkhantsoe I just thought that the word `useage` is a typo 😄

Reply:
Ahh.. I see will update it. 👍